subsys/bluetooth/host/CMakeLists.txt

```diff
@@ -3,32 +3,16 @@
 zephyr_library()
 zephyr_library_link_libraries(subsys__bluetooth)
 
+add_subdirectory_ifdef(CONFIG_BT_BREDR classic)
```

Comment:
An open question: 
Should we use classic or BREDR everywhere?
Right now, we have the Kconfigs called BREDR, but the directory is called classic. 

So the new directory be `bredr` instead? Or should the Kconfig be renamed to `CONFIG_BT_CLASSIC`? 

That way we would be consistent with BT Mesh and BT Audio

Reply:
I would also prefer `BT_CLASSIC`.
We can keep the `BREDR` kconfig as deprecated and make it select `CLASSIC`.